dtfs-central-api/src/services/portal/facility-amendment.upsertPortalFacilityAmendmentDraft.service.test.ts

```diff
@@ -70,6 +88,34 @@ describe('PortalFacilityAmendmentService', () => {
       ).rejects.toThrow(InvalidAuditDetailsError);
     });
 
+    it('should call findOneFacility with the facility id', async () => {
+      // Act
+      await PortalFacilityAmendmentService.upsertPortalFacilityAmendmentDraft({
+        dealId,
+        facilityId,
+        amendment,
+        auditDetails,
+      });
+
+      // Assert
+      expect(mockFindOneFacility).toHaveBeenCalledTimes(1);
+      expect(mockFindOneFacility).toHaveBeenCalledWith(facilityId);
+    });
+
+    it('should call findLatestEligibilityCriteria with the facility type', async () => {
```

Comment:
Can we please add an assertion with facility type `Bond` to assert an empty response back from the EC repo.

Reply:
Good idea - I've now added API tests to check this, so an API test that checks that an error is thrown if there are no ECs for the given facility type.